lib/beacon/boot.ex

```diff
@@ -42,9 +42,33 @@ defmodule Beacon.Boot do
     # TODO: revisit this timeout after we upgrade to Tailwind v4
     Task.await_many(assets, :timer.minutes(5))
 
+    warm_pages(config)
```

Comment:
We can make it run async along with JS and CSS so it loads assets and warm up pages at the same time (one doesn't depend on the other).

```elixir
resources = [
  css,
  js,
  pages
]

await_many(resources)
```